### ISSUE TITLE
ValueError on Empty Input for Save Quote Prompt

### ISSUE THREAD

**Issue by nickdibari** [nickdibari/Get-Quote#8]
When user is prompted to enter a quote to save to the database after script prints the retrieved quotes, the program crashes on an empty input

To Duplicate:
- Run script with any author
- Get to prompt that asks user to select a quote to save
- Press the Enter key without entering a value

Traceback:
```
Traceback (most recent call last):
  File "./getquote.py", line 215, in <module>
    Main()
  File "./getquote.py", line 207, in Main
    SaveQuote(quotes_DB, quotes, author)
  File "./getquote.py", line 71, in SaveQuote
    elif int(choice)<0 or int(choice)>len(quotes):                                                                                                                                             
ValueError: invalid literal for int() with base 10: '' 
```

**Comment by nickdibari** [comment]
Proposed Solution:
Wrap the int( ) call on choice in a try/except block about the check for a proper value in range. Except ValueError, print an appropriate response and handle gracefully

**Comment by nickdibari** [comment]
Fixed with 912cca8
